about.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.15.1/devicon.min.css">
+    <link rel="stylesheet" href="./css/about.css">
+    <title>Save Our Planet Global Summit on Climate Change | About</title>
+</head>
+<body>
+    <header>
+         
+        <div class="menu-container">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+        </div>
+
+        <nav class="mobile-nav">
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#program">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a href="#">SOP Campaign</a></li>
+            </ul>
+        </nav>
```

Comment:
- [ ] When I view the about page on the mobile version, I do not have a link that takes me to the home page on the mobile version. Kindly change
 `<li><a href="about.html">About</a></li>` to `<li><a href="index.html">Home</a></li>` on the mobile menu for the about page. 👍